coil-sample/src/main/java/coil/sample/CoilApplication.kt

```diff
@@ -18,24 +18,24 @@ import okhttp3.Dispatcher
 import okhttp3.OkHttpClient
 import java.io.File
 
-class Application : MultiDexApplication(), ImageLoaderFactory {
+class CoilApplication : Application(), ImageLoaderFactory {
```

Comment:
Why rename the application + inherit from `Application` instead of `MultiDexApplication`?

Reply:
`MultiDexApplication` is required for multidex support for below API 21.

Reply:
Sorry, I forgetd the rule of multidex support for below API 21, reverted